Kickstarter-iOS/DataSources/CategorySelectionDataSource.swift

```diff
@@ -21,24 +21,4 @@ internal final class CategorySelectionDataSource: ValueCellDataSource {
       assertionFailure("Unrecognized (cell, value) combo.")
     }
   }
-
-  public func collectionView(
```

Comment:
In order to work for collectionView and tabelView, I changed the `CategoryCollectionViewSectionHeaderView` to inherit from `UIView` instead and pinned it to the top of the screen. So we no longer need to register a supplementary view.

Reply:
I'm confused, why does this need to be reused? These section headers are just for the collection view - there's no indication that we need section headers for the curated projects screen.

![image](https://user-images.githubusercontent.com/3156796/76121735-573b0000-5fc2-11ea-95c2-738fe81f6f33.png)

Curated projects screen designs:

![image](https://user-images.githubusercontent.com/3156796/76121774-70dc4780-5fc2-11ea-898e-d52e2f1c813e.png)

Reply:
OMG, I though I was changing `CategorySelectionHeaderView` and not  `CategoryCollectionViewSectionHeaderView`! 🤦🏻‍♂️ I reverted that changes back. Sorry for the confusion.